crates/core/src/addon.rs

```diff
@@ -258,9 +258,12 @@ impl Addon {
             let version = package.version.clone();
             let download_url = package.url.clone();
 
-            let date_time = NaiveDateTime::parse_from_str(&package.lastupdate, "%Y-%m-%d")
+            let date_time = NaiveDateTime::parse_from_str(&package.lastupdate, "%Y-%m-%d %H:%M:%S")
                 .map_or(
```

Comment:
Does it make sense to use `Result::map_or_else` here to avoid parsing a second time if it succeeds the first time. My understanding is this will parse twice every time, even if the first is successful.